src/components/UsernameForm/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import withRouter from 'react-router-dom/withRouter';
+import TimeMessage from './TimeMessage';
+import UsernameInput from './UsernameInput';
+import CheckButton from './CheckButton';
+
+class UsernameForm extends Component {
+  static propTypes = {
```

Comment:
This comes up an ESLint error for me, see https://stackoverflow.com/a/43426331/380054 this is similar to the spread operator thing I mentioned above. We have a few options to fix, do you have any preference?

Reply:
Set ESLint to support babel-preset-stage-2, it comes with create-react-app by default.

Reply:
Yeah, based off your comments above too I think this is the way to go 👍